symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -37,25 +47,22 @@ public function registerBundles()
 
     protected function configureContainer(ContainerBuilder $container, LoaderInterface $loader)
     {
-        $container->addResource(new FileResource($this->getProjectDir().'/config/bundles.php'));
+        $container->addResource(new FileResource($this->getBundlesFile()));
         // Feel free to remove the "container.autowiring.strict_mode" parameter
         // if you are using symfony/dependency-injection 4.0+ as it's the default behavior
         $container->setParameter('container.autowiring.strict_mode', true);
         $container->setParameter('container.dumper.inline_class_loader', true);
-        $confDir = $this->getProjectDir().'/config';
```

Comment:
Instead I'd suggest to only replace this change by this:

```diff
-        $confDir = $this->getProjectDir().'/config';
+        $confDir = $this->getConfigDir();
```
Having a single var reduces the amount of calls, and therefore improves performances 😉